product_docs/docs/sql_assess/1/using.mdx

```diff
@@ -0,0 +1,260 @@
+
+   ```text
+   /u01/app/Oracle/diag/diag/rdbms/orcl/ORCL/trace/ORCL_ora_122523.trc
+   ```
+
+9. Copy the trace file and share it with the migration assessment team.
+
+10. In the Oracle client, execute the following statement to stop tracing of the application session:
+
+   ```SQL
+   EXECUTE dbms_system.set_sql_trace_in_session(16, 6590, false);
+   ```
+
+## Using the EDB Query Extractor
+
+The EDB Query Extractor is a SQL script that can be executed via SQL*Plus to extract queries run by the user application from the `V$SQL` system table. The EDB Query Extractor script expects four input parameters:
+
+- Location for output file &mdash; Provide the path where you want the EDB Query Extractor to place the SQL statements extracted from the `V$SQL` table. By default, the trace file generates from the location where SQLPlus was run from.
+
+- The name of the user whose queries are to be extracted &mdash; The script extracts the SQL statements run only by the provided Oracle user. By default, the script uses the current logged-in user.
+
+- The start timestamp of the queries &mdash; Provide the starting timestamp of the queries to extract in the format DD-MM-YYYY HH24:MI:SS. By default, the script uses `01-01-1970`.
+
+- The end timestamp of the queries &mdash; Provide the ending timestamp of the queries to extract in the format DD-MM-YYYY HH24:MI:SS. By default, the script uses the current time.
+
+The following is an example of the EDB Query Extractor input parameters:
+
+```
+SQL>@edb_queries_extractor.sql
+# -- EDB Query Extractor Version 1.0.0 for Oracle Database -- #
+# ----------------------------------------------------------- #
+Caution:
+Script USER must have CONNECT role and SELECT ANY DICTIONARY privilege.
+(To verify granted roles and privileges, run the following commands: SELECT granted_role FROM user_role_privs;  or  SELECT privilege FROM user_sys_privs;)
+Press RETURN to continue ...
+Location for output file (Default current location): /tmp/
+Enter the name of the user whose queries are to be extracted (Default current user): rs2_lin32
+Enter the start timestamp of the queries (DD-MM-YYYY HH24:MI:SS) (Default 01-01-1970): 
+Enter the end timestamp of the queries (DD-MM-YYYY HH24:MI:SS) (Default current time): 
+
+Writing queries to /tmp/RS2_LIN32_230518054450.out
+####################################################################################################
+## EDB Query Extractor Utility. Version: 1.0.0
+##
+## Source Database Version	: Oracle Database 19c Enterprise Edition Release 19.0.0.0.0 - Production
+## Queries Extracted for User	: rs2_lin32 (User Id: 109)
+## Time Duration for Extraction	: 01-01-1970 00:00:00 to 18-05-2023 05:44:50
+## Extracted On			: 18-05-2023 05:44:50
+####################################################################################################
+Disconnected from Oracle Database 19c Enterprise Edition Release 19.0.0.0.0 - Production
+Version 19.3.0.0.0
+
+```
+
```

Comment:
@ccestes We should add a section here for using plain SQL files.  The section will likely not have a lot of steps, but it could provide examples of the accepted content of the scripts.

@gabbasb can you provide a simple set of examples of the plain SQL statements that are valid?

Reply:
Example plain SQL statements that are valid:
`INSERT INTO PLSQLTEST VALUES ('Hello World');

INSERT INTO PLSQLTEST VALUES (:B1 );

SELECT * FROM EMP WHERE JOB = :B1;

DELETE FROM PLSQLTEST;

BEGIN
   PRO_DROP_PRIMARY_KEY('PK_NAME_HERE');
   
   MSCH.INS_LOG_MSG(:msg_number, :msg_proc_id, 'DROP PRIMARY KEY' , :msg_type);
END;
/
`

Reply:
@mlewandowski-edb I added these examples as a subsection of Using the EDB Query Extractor. Should they be in their own section?

Reply:
@ccestes Yes, these examples should be in their own (new) section.  I suggest "Using plain SQL files" as the title of the section.  The section would be at the same level as the "Generating a SQL trace file" and "Using the EDB Query Extractor" sections.